_posts/2020-11-09-pulling-most-recent-files.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title:  "Pull most recent files in R"
+date:   2020-11-09
+categories: jekyll update
+---
+# Pulling most recent files in R
+ ###### by Joel Christian Andrade
+
+Many coding pipelines involve taking raw data files, formulating code and eventually working with a few, if not many, output files.
+
+Take for example the list of files, in my directory `/raw_data` below from the publicly available Apple mobility database tracking movement.
+
+```
+applemobilitytrends_2020-08-29.csv
+applemobilitytrends_2020-09-12.csv
+applemobilitytrends_2020-09-18.csv
+applemobilitytrends_2020-09-25.csv
+applemobilitytrends_2020-09-30.csv
+applemobilitytrends_2020-10-02.csv
+applemobilitytrends_2020-11-08.csv
+```
+
+While retroactive data analysis is useful, the most recent file will include the data of the previous files.  Thus, to retrieve the most up to date information while using R, we must use some functions which are pre-loaded into R.
+
+First, we must elucidate these functions
+* `file.info` = provides some basic information about a file(s)
+ * size
+ * if the file is directory
+ * if the file is an executable
+ * editing times
+   * `mtime` = modification time
+   * `atime` = access time
+   * `ctime` = creation time
+
+
+* `list.files` = list of all files within a directory
+
+* `which.min` or `which.max` = determine minimum or maximum with respect to a given set of variables
+
+Let's take the directory `raw_data` and view various outputs using these functions
```

Comment:
I like how you included a detailed example!

Reply:
thanks!